### ISSUE TITLE
Symbols in LakeBtc are in lower case. This is not consistent with the other exchanges.

### ISSUE THREAD

**Issue by LaurentIsRunning** [ccxt/ccxt#1663]
Symbols in LakeBtc are in lower case. This is not consistent with the other exchanges.

- OS: Windows 10
- Programming Language version: Python
- CCXT version: 1.10.914
- Exchange: lakeBTC
- Method: fetchBalance

**Comment by kroitor** [comment]
```
Last login: Sun Jan 28 15:31:39 on ttys001
mbp:ccxt igorkroitor$ node examples/js/symbols lakebtc
Instantiating lakebtc exchange

Symbols:

limits | precision | tierBased | percentage | taker | maker  | id     | symbol  | base | quote | info           
----------------------------------------------------------------------------------------------------------------
       |           |           |            | 0.002 | 0.0015 | audusd | AUD/USD | AUD  | USD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | baceth | BAC/ETH | BAC  | ETH   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcaud | BTC/AUD | BTC  | AUD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btccad | BTC/CAD | BTC  | CAD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcchf | BTC/CHF | BTC  | CHF   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btceur | BTC/EUR | BTC  | EUR   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcgbp | BTC/GBP | BTC  | GBP   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcjpy | BTC/JPY | BTC  | JPY   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcngn | BTC/NGN | BTC  | NGN   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcnzd | BTC/NZD | BTC  | NZD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcsgd | BTC/SGD | BTC  | SGD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | btcusd | BTC/USD | BTC  | USD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | ethbtc | ETH/BTC | ETH  | BTC   | [object Object]
       |           |           |            | 0.002 | 0.0015 | eurusd | EUR/USD | EUR  | USD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | gbpusd | GBP/USD | GBP  | USD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | ltcbtc | LTC/BTC | LTC  | BTC   | [object Object]
       |           |           |            | 0.002 | 0.0015 | nzdusd | NZD/USD | NZD  | USD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | usdcad | USD/CAD | USD  | CAD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | usdchf | USD/CHF | USD  | CHF   | [object Object]
       |           |           |            | 0.002 | 0.0015 | usdhkd | USD/HKD | USD  | HKD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | usdjpy | USD/JPY | USD  | JPY   | [object Object]
       |           |           |            | 0.002 | 0.0015 | usdngn | USD/NGN | USD  | NGN   | [object Object]
       |           |           |            | 0.002 | 0.0015 | usdsgd | USD/SGD | USD  | SGD   | [object Object]
       |           |           |            | 0.002 | 0.0015 | xrpbtc | XRP/BTC | XRP  | BTC   | [object Object]

---------------------------------------------------------------

Currencies:

id  | code | precision
----------------------
AUD | AUD  | 8        
BAC | BAC  | 8        
BTC | BTC  | 8        
CAD | CAD  | 8        
CHF | CHF  | 8        
ETH | ETH  | 8        
EUR | EUR  | 8        
GBP | GBP  | 8        
HKD | HKD  | 8        
JPY | JPY  | 8        
LTC | LTC  | 8        
NGN | NGN  | 8        
NZD | NZD  | 8        
SGD | SGD  | 8        
USD | USD  | 8        
XRP | XRP  | 8        

---------------------------------------------------------------
lakebtc has 24 symbols and 16 currencies

mbp:ccxt igorkroitor$ 
```

**Comment by kroitor** [comment]
```
mbp:ccxt igorkroitor$ php -f examples/php/symbols.php lakebtc
Instantiating lakebtc exchange exchange
lakebtc has 24 symbols: AUD/USD, BAC/ETH, BTC/AUD, BTC/CAD, BTC/CHF, BTC/EUR, BTC/GBP, BTC/JPY, BTC/NGN, BTC/NZD, BTC/SGD, BTC/USD, ETH/BTC, EUR/USD, GBP/USD, LTC/BTC, NZD/USD, USD/CAD, USD/CHF, USD/HKD, USD/JPY, USD/NGN, USD/SGD, XRP/BTC
+--------+---------+------+-------+-------+--------+-----------+--------+
| id     | symbol  | base | quote | taker | maker  | precision | limits |
+--------+---------+------+-------+-------+--------+-----------+--------+
| btcusd | BTC/USD | BTC  | USD   | 0.002 | 0.0015 | []        | []     |
| btceur | BTC/EUR | BTC  | EUR   | 0.002 | 0.0015 | []        | []     |
| btcjpy | BTC/JPY | BTC  | JPY   | 0.002 | 0.0015 | []        | []     |
| btcgbp | BTC/GBP | BTC  | GBP   | 0.002 | 0.0015 | []        | []     |
| btcaud | BTC/AUD | BTC  | AUD   | 0.002 | 0.0015 | []        | []     |
| btccad | BTC/CAD | BTC  | CAD   | 0.002 | 0.0015 | []        | []     |
| btcsgd | BTC/SGD | BTC  | SGD   | 0.002 | 0.0015 | []        | []     |
| btcchf | BTC/CHF | BTC  | CHF   | 0.002 | 0.0015 | []        | []     |
| btcnzd | BTC/NZD | BTC  | NZD   | 0.002 | 0.0015 | []        | []     |
| btcngn | BTC/NGN | BTC  | NGN   | 0.002 | 0.0015 | []        | []     |
| eurusd | EUR/USD | EUR  | USD   | 0.002 | 0.0015 | []        | []     |
| gbpusd | GBP/USD | GBP  | USD   | 0.002 | 0.0015 | []        | []     |
| usdjpy | USD/JPY | USD  | JPY   | 0.002 | 0.0015 | []        | []     |
| usdhkd | USD/HKD | USD  | HKD   | 0.002 | 0.0015 | []        | []     |
| usdcad | USD/CAD | USD  | CAD   | 0.002 | 0.0015 | []        | []     |
| usdsgd | USD/SGD | USD  | SGD   | 0.002 | 0.0015 | []        | []     |
| audusd | AUD/USD | AUD  | USD   | 0.002 | 0.0015 | []        | []     |
| nzdusd | NZD/USD | NZD  | USD   | 0.002 | 0.0015 | []        | []     |
| usdchf | USD/CHF | USD  | CHF   | 0.002 | 0.0015 | []        | []     |
| usdngn | USD/NGN | USD  | NGN   | 0.002 | 0.0015 | []        | []     |
| ethbtc | ETH/BTC | ETH  | BTC   | 0.002 | 0.0015 | []        | []     |
| ltcbtc | LTC/BTC | LTC  | BTC   | 0.002 | 0.0015 | []        | []     |
| xrpbtc | XRP/BTC | XRP  | BTC   | 0.002 | 0.0015 | []        | []     |
| baceth | BAC/ETH | BAC  | ETH   | 0.002 | 0.0015 | []        | []     |
+--------+---------+------+-------+-------+--------+-----------+--------+

mbp:ccxt igorkroitor$ 
```

**Comment by kroitor** [comment]
- https://github.com/ccxt/ccxt/wiki/Manual#markets
- https://github.com/ccxt/ccxt/wiki/Manual#symbols-and-market-ids
- https://github.com/ccxt/ccxt/wiki/Manual#naming-consistency
- https://github.com/ccxt/ccxt/wiki/Manual#consistency-of-base-and-quote-currencies

**Comment by kroitor** [comment]
Let us know if the above does not help.

**Comment by LaurentIsRunning** [comment]
I had to change line 96 in lakebtc to correct that:
result[currency**.upper()**] = account

**Comment by kroitor** [comment]
@LaurentIsRunning oh, ok, thx for pointing that out, will fix it shortly.

**Comment by kroitor** [comment]
Fixed in 1.10.939. Thx again!

**Comment by LaurentIsRunning** [comment]
You're welcome !
